src/main/java/de/undercouch/gradle/tasks/download/DownloadAction.java

```diff
@@ -168,7 +171,23 @@ private void execute(URL src, HttpClientFactory clientFactory) throws IOExceptio
                         + "progress will not be displayed.");
             }
         }
-        
+
+        if ("file".equals(src.getProtocol())) {
```

Comment:
Sure, I can do that, just initially I wanted to make my PR have as little new code as possible.  This makes sense.